modules/src/events.rs

```diff
@@ -261,32 +250,31 @@ impl fmt::Display for IbcEvent {
-
-            IbcEvent::OpenInitChannel(ev) => write!(f, "OpenInitChannelEv({:?})", ev),
-            IbcEvent::OpenTryChannel(ev) => write!(f, "OpenTryChannelEv({:?})", ev),
-            IbcEvent::OpenAckChannel(ev) => write!(f, "OpenAckChannelEv({:?})", ev),
-            IbcEvent::OpenConfirmChannel(ev) => write!(f, "OpenConfirmChannelEv({:?})", ev),
-            IbcEvent::CloseInitChannel(ev) => write!(f, "CloseInitChannelEv({})", ev),
-            IbcEvent::CloseConfirmChannel(ev) => write!(f, "CloseConfirmChannelEv({:?})", ev),
-
-            IbcEvent::SendPacket(ev) => write!(f, "SendPacketEv({})", ev),
-            IbcEvent::ReceivePacket(ev) => write!(f, "ReceivePacketEv({})", ev),
-            IbcEvent::WriteAcknowledgement(ev) => write!(f, "WriteAcknowledgementEv({})", ev),
-            IbcEvent::AcknowledgePacket(ev) => write!(f, "AcknowledgePacketEv({})", ev),
-            IbcEvent::TimeoutPacket(ev) => write!(f, "TimeoutPacketEv({})", ev),
-            IbcEvent::TimeoutOnClosePacket(ev) => write!(f, "TimeoutOnClosePacketEv({})", ev),
-
-            IbcEvent::Empty(ev) => write!(f, "EmptyEv({})", ev),
-            IbcEvent::ChainError(ev) => write!(f, "ChainErrorEv({})", ev),
+            IbcEvent::CreateClient(ev) => write!(f, "CreateClient({})", ev),
+            IbcEvent::UpdateClient(ev) => write!(f, "UpdateClient({})", ev),
+            IbcEvent::UpgradeClient(ev) => write!(f, "UpgradeClient({:?})", ev),
+            IbcEvent::ClientMisbehaviour(ev) => write!(f, "ClientMisbehaviour({:?})", ev),
+
+            IbcEvent::OpenInitConnection(ev) => write!(f, "OpenInitConnection({:?})", ev),
+            IbcEvent::OpenTryConnection(ev) => write!(f, "OpenTryConnection({:?})", ev),
+            IbcEvent::OpenAckConnection(ev) => write!(f, "OpenAckConnection({:?})", ev),
+            IbcEvent::OpenConfirmConnection(ev) => write!(f, "OpenConfirmConnection({:?})", ev),
+
+            IbcEvent::OpenInitChannel(ev) => write!(f, "OpenInitChannel({:?})", ev),
+            IbcEvent::OpenTryChannel(ev) => write!(f, "OpenTryChannel({:?})", ev),
+            IbcEvent::OpenAckChannel(ev) => write!(f, "OpenAckChannel({:?})", ev),
+            IbcEvent::OpenConfirmChannel(ev) => write!(f, "OpenConfirmChannel({:?})", ev),
+            IbcEvent::CloseInitChannel(ev) => write!(f, "CloseInitChannel({})", ev),
+            IbcEvent::CloseConfirmChannel(ev) => write!(f, "CloseConfirmChannel({:?})", ev),
+
+            IbcEvent::SendPacket(ev) => write!(f, "SendPacket({})", ev),
+            IbcEvent::ReceivePacket(ev) => write!(f, "ReceivePacket({})", ev),
+            IbcEvent::WriteAcknowledgement(ev) => write!(f, "WriteAcknowledgement({})", ev),
+            IbcEvent::AcknowledgePacket(ev) => write!(f, "AcknowledgePacket({})", ev),
+            IbcEvent::TimeoutPacket(ev) => write!(f, "TimeoutPacket({})", ev),
+            IbcEvent::TimeoutOnClosePacket(ev) => write!(f, "TimeoutOnClosePacket({})", ev),
```

Comment:
Intersting. I wonder why are these lines marked as changed since I don't think there should be any diff.

Reply:
This removes the unnecessary "Ev" suffixes from the text repr, as discussed in https://github.com/informalsystems/ibc-rs/pull/1909#discussion_r812036743